server/middleware/auth.js

```diff
@@ -0,0 +1,13 @@
+const jwt = require('jsonwebtoken')
+
+module.exports = function(req, res, next) {
+    const authToken = req.header("user-auth-token");
+
+    if (!authToken) {
+        return res.status(401).send('User not authorized!')
+    }
+
+    const userDetails = jwt.verify(authToken, 'awognawole3280843halegn');
```

Comment:
Later on let's save the key on secrets. For sure, we'll have to erase the testing data and set up a new key.